src/DNS/Server.php

```
@@ -173,496 +161,103 @@ public function start(): void
+                        [],
+                        []
+                    );
+                }
+                $this->duration?->record(microtime(true) - $resolveStart, [
+                    'phase' => 'resolve'
+                ]);
 
-                    $processingTime = (microtime(true) - $startTime);
-                    $this->resolveDuration?->record($processingTime, ['type' => $type]);
-                    Console::info("[PACKET] Processing completed in {$processingTime}s");
+                if (empty($response->answers)) {
```

PHPMD (2.15.0)

Avoid unused local variables such as '$response'. (undefined)

(UnusedLocalVariable)